.github/workflows/docker-build-push-google.yaml

```diff
@@ -77,6 +82,7 @@ jobs:
       packages: read
       id-token: write
     env:
+      DOCKER_AUTH: ${{ secrets.docker-username && secrets.docker-token && 'true' || 'false' }}
```

Comment:
I would have expected that `${{ secrets.docker-username && secrets.docker-token || false }}` would be enough since the first part of the expression should anyway "resolve" to `true` as long as both secrets exist, no?